src/count-blog-domain.js

```diff
@@ -0,0 +1,52 @@
+/* This file contains the code for counting the blog posts from different domains.
+ * The module receives an array of blog urls and returns an array of json object in format
+ * {name: domainName, count: number of frequency}
+*/
+
+//  import url library
+const urls = require('url');
```

Comment:
drop the `s` from `urls`, it should be `url`

Reply:
@humphd I tried to drop the ```s``` from ```const urls = require('url');```
it will have error in  ```feedUrls.slice(1).forEach(({ url }) => {``` cause they are using the same name. The ```{url}``` using in ```feedUrls.slice(1).forEach(({ url }) => {``` is the object property name so I cannot change this one.

Reply:
In your loop below, rename it to `feedUrl` instead of `url` to better match with the context of the code.